NAMESPACE

```diff
@@ -26,10 +28,11 @@ export(deepstate_logcheck)
 export(deepstate_logtest)
 export(deepstate_make_run)
 export(deepstate_pkg_create)
+export(deepstate_read_valgrind_xml)
 export(deepstate_test)
 export(deepstate_tests_fuzz)
 export(deepstate_user_error_display)
-export(deepstate_xmlog)
+export(get_string)
```

Comment:
please do not export any functions that you do not want users to run

Reply:
not exporting the functions anymore